tests/src/jest.test.ts

```diff
@@ -0,0 +1,16 @@
+describe("jest", () => {
+    /**
+     * This test will verify if globals are configured correctly
```

Comment:
I mean sure, but this feels like testing if a new-born dog has legs.  It's going to be true unless something in the setup process goes horribly wrong.

Reply:
Agreed, this test was more simple test-example than anything else. However, it does somewhat validate the integrity of the tests that are run. So a bit of both really.